products/sle/main.pm

```diff
@@ -1540,8 +1562,13 @@ elsif (get_var('HPC')) {
         load_reboot_tests();
     }
     else {
-        loadtest 'boot/boot_to_desktop';
-        loadtest 'hpc/setup_network' if check_var('NICTYPE', 'tap');
+        if (get_var('CLUSTER_NODES')) {
+            load_cluster_boot();
+        }
+        else {
+            loadtest 'boot/boot_to_desktop';
+            loadtest 'hpc/setup_network' if check_var('NICTYPE', 'tap');
```

Comment:
I think you can drop it , because it was used for cases which will currently use your approach. also you can drop 'hpc/setup_network' as we will not use it anymore

Reply:
it is there for backward compatibility, so far ganglia test needs it